dagobah/core/dagobah.py

```diff
@@ -100,9 +100,9 @@ def _add_job_from_spec(self, job_json, use_job_id=True):
             job.schedule(job_json['cron_schedule'])
 
         for task in job_json.get('tasks', []):
-            # If it is a jobtask, it will have a job_id
-            if task.get('job_id'):
-                self.add_jobtask_to_job(job, task['job_id'], str(task['name']))
```

Comment:
a task cant exist without a name, that is it's ID. Same basically with a job's name (although jobs have unique IDs as well), if it didn't have a name, there is a lot of dagobah that would break